dist/components/ratio/_ratio.scss

```diff
@@ -32,10 +32,6 @@
 // ---
 //
 // Should be the sole child of a Ratio container.
-//
-// TODO: There are issues in iOS when placing <iframe> or <object> elements
```

Comment:
No, but I created an issue for this https://github.com/mobify/stencil/issues/131